content/shmem_wait.tex

```diff
@@ -30,7 +30,8 @@
 \begin{apiarguments}
 
 \apiargument{OUT}{ivar}{A remotely accessible integer variable that is being updated
-    by another \ac{PE}.  If you are using \CorCpp, the type of \VAR{ivar} should
+    by another \ac{PE} \newtext{or the same \ac{PE} in the case of multi-threaded
```

Comment:
There also be a unlikely case where the PE/thread calling shmem_wait is the one updating the variable (before it calls shmem_wait). This is unlikely but should not be invalid. Should we just say "A remotely accessible integer variable."?